eppo_client/client.py

```diff
@@ -31,9 +34,47 @@ def __init__(
         )
         self.__poller.start()
 
+    def get_string_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[str]:
+        assigned_variation = self.get_assignment_variation(subject_key, flag_key, subject_attributes, VariationType.STRING)
+        return assigned_variation.typedValue if assigned_variation is not None else assigned_variation
+
+    def get_numeric_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[Number]:
+        assigned_variation = self.get_assignment_variation(subject_key, flag_key, subject_attributes, VariationType.NUMERIC)
+        return assigned_variation.typedValue if assigned_variation is not None else assigned_variation
+
+    def get_boolean_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[bool]:
+        assigned_variation = self.get_assignment_variation(subject_key, flag_key, subject_attributes, VariationType.BOOLEAN)
+        return assigned_variation.typedValue if assigned_variation is not None else assigned_variation
+
+    def get_parsed_json_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[Dict[Any, Any]]:
+        assigned_variation = self.get_assignment_variation(subject_key, flag_key, subject_attributes, VariationType.JSON)
+        return assigned_variation.typedValue if assigned_variation is not None else assigned_variation
+
+    def get_json_string_assignment(
+        self, subject_key: str, flag_key: str, subject_attributes=dict()
+    ) -> Optional[str]:
+        assigned_variation = self.get_assignment_variation(subject_key, flag_key, subject_attributes, VariationType.JSON)
+        return assigned_variation.value if assigned_variation is not None else assigned_variation
```

Comment:
Assumption that `assigned_variation.value` will always be the string-type equivalent of `assigned_variation.typedValue` and so I can return it instead of decoding the `typedValue` to a string

Their inter-convertibility is also verified in the `is_expected_type` method